### DISCUSSION TITLE
Error: zsh: parse error near `\n'

### DISCUSSION THREAD

**Discussion by salzlete** [karloskalcium/md2enex#11] [General]
Hey,
thanks for the great script!
I am definitely not an expert and can only copy-paste what the instructions say without understanding exactly what I'm doing.

In the last step I get the following error:
zsh: parse error near `\n'

Any idea how I can fix that?

Thanks in advance!

**Comment by karloskalcium** [comment]
hi I changed this to a discussion - can you describe more which step this failed on, and what command you were running when it happened?  @salzlete
